src/Avalonia.Diagnostics/Diagnostics/Views/MainWindow.xaml

```diff
@@ -4,17 +4,15 @@
         xmlns:diag="clr-namespace:Avalonia.Diagnostics"
         xmlns:default="using:Avalonia.Themes.Default"
         Title="Avalonia DevTools"
-        x:Class="Avalonia.Diagnostics.Views.MainWindow">
+        x:Class="Avalonia.Diagnostics.Views.MainWindow"
+        Theme="{StaticResource {x:Type Window}}">
```

Comment:
@grokys without this line devtools has empty window without any template. It also works, if I replace Window ControlTheme with old Window Style.

Reply:
This happens because `TopLevel` [styles itself in the constructor](https://github.com/AvaloniaUI/Avalonia/blob/master/src/Avalonia.Controls/TopLevel.cs#L185) and at this point the `Window.Styles` containing the theme haven't been assigned.

Explicitly setting the `Theme` property causes the styling to be reset and styling applied on next layout.

`TopLevel` styling itself in its constructor is terrible behavior that I hacked together back in 2015 and something we need to change (I tried, unfortunately it's not that easy and will break stuff) so I think this hackfix is the best we can do for now.